tutorials/pretrain_redpajama.md

```diff
@@ -30,7 +30,7 @@ You can download the data using git lfs:
 
 ```bash
 # Make sure you have git-lfs installed (https://git-lfs.com):
-git lfs install
+sudo apt install git-lfs
```

Comment:
After `git lfs` is installed shouldn't it be configure for the account by running `git lfs install`?
I believe the latter command changes global .gitconfig file.
<img width="525" alt="Screenshot 2023-11-28 at 11 29 24 AM" src="https://github.com/Lightning-AI/lit-gpt/assets/58434077/177ca404-b3c6-4f65-8c68-82570e3829f2">

So I believe it's a two-command operation:
1. sudo apt install git-lfs
2. git lfs install